aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -1,54 +1,28 @@
-| ----------------------------- | ------------------------------------------- |
-| Who they hear                 | What they hear                              |
-| Personality                   | Attitude                                    |
-| How you are doing the writing | Who they hear                               |
-| Style                         | How your message comes across               |
-| What makes us uniquely us     | How we communicate to our specific audience |
-
-### Voice
-
-Voice is who the reader "hears" delivering messaging; embedded within it is the
-worldview of the author. Voice can be spoken, witty, friendly, formal, etc.
-At HPE, the desire is for HPE's voice to be perceived as intelligent, experienced,
-objective, inclusive, and trustworthy &mdash; supporting HPE's worldview which
-believes in accelerating what's next, making bold moves, the power of _yes we can_, and
-being a force for good.
-
-A person's voice typically does not change once established. If HPE's voice is
-inconsistent, people will notice. For example, people notice the difference
-between writing academically when compared to sounding like a 5-year old. This
-kind of inconsistency can result in lack of trust.
-
-### Tone
-
-Tone has to do with what is conveyed or how it comes across. It can express seriousness, urgency, playfulness, objectivity, intimacy, condescension, etc.
-
-Unlike voice, tone can change based on the target audience, context, and subject
-matter. It is situational. For example, tone is very different when giving
-directions during an emergency compared to when ordering a pizza.
-
-Because tone is situational, it may be tweaked to appropriately fit the context
-within an HPE application. While tone may change, the message should be consistently
-delivered in [HPE's voice](#voice) supporting the perception of intelligence,
-experience, objectivity, and inclusivity.
-
-### Point of View
+The HPE Design System employs text-based content that cultivates an exceptional user experience.
+<!-- The above sentence will be the leading line and moved to the Voice and Tone card in Foundations once approved -->
+
+The user experience is built upon our content. Every word we write is a carefully-considered concept that voices our values: 
+
+- **Open**: We foster inclusive experiences that are personal, accessible, and transparent.
+- **Courageous**: We empower users with decisive and experienced guidance.
+- **Inspired**: We share our drive of 'yes we can' with users.
+
+Refer to the guidance for [HPE's Voice](https://brandcentral.hpe.com/brand-central/content/voice) for more information.
+
+## Using Our Voice
+
+We exercise empathy in our voice through situational tones that speak to each context, audience, and subject matter. We can be a force for good by creating content that is:
+
+- Active
+- Positive
+- Objective
+- Plain English
+- [Third-person point of view](#point-of-view)
```

Comment:
- I don't think its necessary to link 'Third-person point of view' since the following section addresses this

- Third-person point of view seems to contradict "empathy." With regards to our reasoning for using a third-person point of view: should we focus more on objectivity in tone? Or messaging that is fact based and without opinions or judgement?

Reply:
Both good points!

I'm going to remove 'Third-person point of view' for now and add try out some options in line with your suggestions. The 'Point of View' section as a whole needs some reflection on my part.